Geometry/HGCalMapping/plugins/alpaka/HGCalMappingCellESProducer.cc

```diff
@@ -41,9 +45,49 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         desc.add<std::vector<std::string> >("filelist", std::vector<std::string>({}))
             ->setComment("list of files with the readout cells of each module");
         desc.add<edm::ESInputTag>("cellindexer", edm::ESInputTag(""))->setComment("Dense cell index tool");
+        desc.add<edm::FileInPath>(
+                "offsetfile",
+                edm::FileInPath("Geometry/HGCalMapping/data/CellMaps/calibration_to_surrounding_offsetMap.txt"))
+            ->setComment("file containing the offsets between calibration and surrounding cells");
         descriptions.addWithDefaultLabel(desc);
       }
 
+      std::map<std::tuple<std::string, int, int>, int> makeOffsetMap(edm::FileInPath input_offsetfile) {
+        std::map<std::tuple<std::string, int, int>, int> offsetMap;
+        const auto& offsetfile = input_offsetfile.fullPath();
+        std::ifstream stream_offsetfile(offsetfile);
+        std::string line;
+        // Skip the first line (description of column content)
+        std::getline(stream_offsetfile, line);
```

Comment:
instead of doing this you could perhaps re-use 
```
::hgcal::mappingtools::HGCalEntityList pmap;
edm::FileInPath fip(offsetfile);
pmap.buildFrom(fip.fullPath());
```          
as done below around L104